src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProject.cs

```diff
@@ -268,16 +268,16 @@ private static IEnumerable<PackageReference> GetPackageReferences(IEnumerable<Li
                 // Install the package to all frameworks.
                 var configuredProject = await _unconfiguredProject.GetSuggestedConfiguredProjectAsync();
 
-                var result = await configuredProject?
+                var result = await configuredProject
```

Comment:
:memo: It appears the previous code intended for `result` to have a nullable value, with null being assigned when `configuredProject` was null. However, at runtime the use of await would actually result in a `NullReferenceException` instead. Since the code was apparently not failing, I updated this on the assumption that `configuredProject` is never null.